### PR TITLE
feat: update README.md and remove a number of broken links

### LINKED ISSUE #473: Clean up repository README.md
README file has a number of broken links
- references to master vs main branches
- reference to EdgeX Chat which no longer exists (vs Slack)
- UserGuide page reference doesn't exist
- VMWare zoom site not needed

### PR DESCRIPTION
**Pull request by badboy-huaqiao**
close #473 
Signed-off-by: Huaqiao <huaqiaoz@vmware.com>

If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-ui-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [ ] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,36 +59,22 @@ make docker run_docker
...
 
 ### Snap 
 
 EdgeX UI is also available as a snap package. For more details on the snap, including how to install it, please refer to [EdgeX UI Snap](https://github.com/edgexfoundry/edgex-ui-go/tree/main/snap/README.md)
 
 ## Community
 - EdgeXFoundry Chinese Club: https://www.edgexfoundry.club
-- Chat: https://chat.edgexfoundry.org/home
-- Mainling lists: https://lists.edgexfoundry.org/mailman/listinfo
+- Chat: https://edgex.slack.com
```
Comment (bnevis-i) [inline README.md:70]: Please add join link: https://slack.edgexfoundry.org/

**Review by bnevis-i — APPROVED** [review]
LGTM, but would like an additional link added.  Also mention which channel the UI discussion is on.

File: README.md (hunk 1 above)
Comment (badboy-huaqiao) [reply README.md:70]: what's  different from this link https://edgex.slack.com

**Comment by badboy-huaqiao** [comment]
> LGTM, but would like an additional link added. Also mention which channel the UI discussion is on.

it's good, the GUI channel's name is `edgex-ui`, it's link I got is https://edgexfoundry.slack.com/archives/CKV38P9LN, do you think it's ok?

File: README.md (hunk 1 above)
Comment (bnevis-i) [reply README.md:70]: slack.edgexfoundry.org issues you and invite to join the slack project.

edgex.slack.com allows you to log in to slack from the web ui once you are already joined.

Comment (badboy-huaqiao) [reply README.md:70]: ok, thanks, I will update it

**Comment by bnevis-i** [comment]
> it's good, the GUI channel's name is `edgex-ui`, it's link I got is https://edgexfoundry.slack.com/archives/CKV38P9LN, do you think it's ok?

I think you should just say "(#edgex-ui channel)" rather than posting the hyperlink.

**Comment by badboy-huaqiao** [comment]
> > it's good, the GUI channel's name is `edgex-ui`, it's link I got is https://edgexfoundry.slack.com/archives/CKV38P9LN, do you think it's ok?
> 
> I think you should just say "(#edgex-ui channel)" rather than posting the hyperlink.

ok, thanks for your advice, hold on, I will update it

File: README.md (hunk 1 above)
Comment (badboy-huaqiao) [reply README.md]: @bnevis-i it's done, please review it again

**Review by bnevis-i — APPROVED** [review]

**Pull request merged**